terraform/20-app/main.tf

```diff
@@ -28,3 +35,53 @@ terraform {
     key            = "app/state.tfstate"
   }
 }
+
+module "cognito" {
+  source = "../modules/cognito"
+
+  user_pool_name    = "app-${terraform.workspace}-user-pool"
```

Comment:
I think we should stick to the naming convention used across the board:
```
${local.prefix}
```
You'd likely have to pass that into the module as a parameter tho cause the module wouldn't have access to the vars from there

Reply:
Amended to `${local.prefix}`